python/VERSION

```diff
@@ -0,0 +1 @@
+0.9.0
```

Comment:
This VERSION file is used by other `setup.py` and `Dockerfile` across the KServe python package.